### ISSUE TITLE
bikeshedding: the example code

### ISSUE THREAD

**Issue by villares** [berinhard/pyp5js#163]
At this point we have, as our first example something like this (actually this is from the docs, at the demo the variable is `radius`, and that bothers me a bit):
```python
def setup():
    createCanvas(200, 200)
    background(160)

def draw():
    fill('blue')
    background(200)
    r = sin(frameCount / 60) * 50 + 50
    ellipse(100, 100, r, r)
```
My proposed change:

```python
def setup():
    createCanvas(200, 200)

def draw():
    background(200)
    diameter = sin(frameCount / 60) * 50 + 50
    fill('blue')
    ellipse(100, 100, diameter, diameter)
```

**Comment by berinhard** [comment]
Feel free to open a PR

**Comment by berinhard** [comment]
I mean, I have no strong opinions about this. So, I won't be against changing this, although is highly unlikely that I'll spare some time to do so. A PR can be very handy =]

**Comment by berinhard** [comment]
Fixed by #168
